Steer/DigitizerWorkflow/src/CTPDigitWriterSpec.h

```diff
@@ -0,0 +1,34 @@
+//
```

Comment:
License missing.

I am asking detectors to move the digit-writers to <detector>/workflow, so it can be used also to write the digits from raw data decoding (but with `o2_` added in front of the name, e.g.  `o2_ctpdigits.root` in order to not overwrite simulated digits, this requires one more option indicating decoded vs simulated input, see e.g. https://github.com/AliceO2Group/AliceO2/blob/f1846f374c44d086c80a421f9467b2e2099472bb/Detectors/FIT/FT0/workflow/src/FT0DigitWriterSpec.cxx#L48-L66)

Reply:
I added CTP/workflow and CTPDigitWriter there in analogy with T0. For my understanding Steer/workflow is not  used if CTP/workflow is available ?

Reply:
Historically,  the DigitWriters were added in the Steer/Digitization to write the result of digitization. 
But then, when it came raw data decoding and storing decoded digits, people started to create almost exact clone of the DigitWriter in the Detectors/XXX/workflow. So, I am encouraging developers to have only 1 DigitWriter and to have it in detector's workflows directory, and to use it  for digits from simulation and raw data decoding (with difference in default output file names mentioned above).